install.rb

```diff
@@ -294,6 +308,12 @@ def prepare_installation
     bindir = RbConfig::CONFIG['bindir']
   end
 
+  if not InstallOptions.localedir.nil?
+    localedir = InstallOptions.localedir
+  else
+    localedir = "/opt/puppetlabs/puppet/share/puppet/locale"
```

Comment:
I assume folder will contain the per-locale localised strings?  I imagine this would be the installation path of Puppet `\locales`  e.g. `C:\Program Files\Puppet Labs\Puppet\puppet\locales`